rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -463,134 +480,133 @@ namespace rocal{
-        m.def("COCOReader", &rocalCreateCOCOReader, py::return_value_policy::reference);
+        m.def("cocoReader", &rocalCreateCOCOReader, py::return_value_policy::reference);
         // rocal_api_meta_data.h
-        m.def("RandomBBoxCrop", &rocalRandomBBoxCrop);
-        m.def("BoxEncoder",&rocalBoxEncoder);
-        // m.def("BoxIOUMatcher", &rocalBoxIOUMatcher);
+        m.def("randomBBoxCrop", &rocalRandomBBoxCrop);
+        m.def("boxEncoder", &rocalBoxEncoder);
+        // m.def("BoxIOUMatcher", &rocalBoxIOUMatcher);  // Will be enabled when IOU matcher changes are introduced in C++
         m.def("getImgSizes", [](RocalContext context, py::array_t<int> array) {
             auto buf = array.request();
-            int* ptr = (int*) buf.ptr;
+            int* ptr = static_cast<int *>(buf.ptr);
             rocalGetImageSizes(context, ptr);
         });
         // rocal_api_parameter.h
-        m.def("setSeed",&rocalSetSeed);
-        m.def("getSeed",&rocalGetSeed);
-        m.def("CreateIntUniformRand",&rocalCreateIntUniformRand);
-        m.def("CreateFloatUniformRand",&rocalCreateFloatUniformRand);
-        m.def("CreateIntRand",[](std::vector<int> values, std::vector<double> frequencies){
+        m.def("setSeed", &rocalSetSeed);
+        m.def("getSeed", &rocalGetSeed);
+        m.def("createIntUniformRand", &rocalCreateIntUniformRand, py::return_value_policy::reference);
+        m.def("createFloatUniformRand", &rocalCreateFloatUniformRand, py::return_value_policy::reference);
+        m.def("createIntRand", [](std::vector<int> values, std::vector<double> frequencies){
             return rocalCreateIntRand(values.data(), frequencies.data(), values.size());
-        });
-        m.def("CreateFloatRand",&rocalCreateFloatRand);
-        m.def("CreateIntParameter",&rocalCreateIntParameter);
-        m.def("CreateFloatParameter",&rocalCreateFloatParameter);
-        m.def("UpdateIntRand", &rocalUpdateIntUniformRand);
-        m.def("UpdateFloatRand", &rocalUpdateFloatUniformRand);
-        m.def("UpdateIntParameter", &rocalUpdateIntParameter);
-        m.def("UpdateFloatParameter", &rocalUpdateFloatParameter);
-        m.def("GetIntValue",&rocalGetIntValue);
-        m.def("GetFloatValue",&rocalGetFloatValue);
+        }, py::return_value_policy::reference);
+        m.def("createFloatRand", &rocalCreateFloatRand, py::return_value_policy::reference);
+        m.def("createIntParameter", &rocalCreateIntParameter, py::return_value_policy::reference);
+        m.def("createFloatParameter", &rocalCreateFloatParameter, py::return_value_policy::reference);
+        m.def("updateIntRand", &rocalUpdateIntUniformRand);
+        m.def("updateFloatRand", &rocalUpdateFloatUniformRand);
+        m.def("updateIntParameter", &rocalUpdateIntParameter);
+        m.def("updateFloatParameter", &rocalUpdateFloatParameter);
+        m.def("getIntValue", &rocalGetIntValue);
```

Comment:
Why some of these are in camelcase and Brightness , ImageDecoder uses pascal case?

Reply:
Changed all pybind functions to camelCase and changed the same in other files